### ISSUE TITLE
Consider warning about missing definition of an IDL method/attribute

### ISSUE THREAD

**Issue by Ms2ger** [tabatkins/bikeshed#1431]
That would have caught https://github.com/WebAssembly/spec/issues/840, for example.

**Comment by tabatkins** [comment]
Yeah, this sounds reasonable. Are there ever *any* cases for not providing a dfn? Methods, sure, but is there ever any reason to let an attribute be defined simply by the IDL?

I don't *think* there is, but I'm not certain.

**Comment by Ms2ger** [comment]
I guess [`[Default]`](https://heycam.github.io/webidl/#Default) would count, but I hope that's not too hard to catch.

**Comment by tabatkins** [comment]
I didn't even know [Default] existed. But yeah, it's not hard to catch.
